reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md

```diff
@@ -207,18 +223,19 @@ Accept wildcard characters: True
 
 ### -CommandType
 
-Exports only the specified types of command objects.
-Use "CommandType" or its alias, "Type".
+Exports only the specified types of command objects. Use **CommandType** or its alias, **Type**.
 
-Valid values are:
+The acceptable values for this parameter are as follows:
 
-- Alias: All Windows PowerShell aliases in the current session.
-- All: All command types. It is the equivalent of "get-command *".
-- Application: All files other than Windows PowerShell files in paths listed in the Path environment variable ($env:path), including .txt, .exe, and .dll files.
-- Cmdlet: The cmdlets in the current session. "Cmdlet" is the default.
-- ExternalScript: All .ps1 files in the paths listed in the Path environment variable ($env:path).
-- Filter and Function: All Windows PowerShell functions.
-- Script: Script blocks in the current session.
+- Alias. All PowerShell aliases in the current session.
+- All. All command types. It is the equivalent of `Get-Command -Name *`.
+- Application. All files other than PowerShell files in paths listed in the Path environment
+  variable (`$env:path`), including .txt, .exe, and .dll files.
+- Cmdlet. The cmdlets in the current session. Cmdlet is the default.
+- ExternalScript. All .ps1 files in the paths listed in the Path environment variable
+  (`$env:path`).
+- Filter and Function. All PowerShell functions.
+- Script. Script blocks in the current session.
```

Comment:
Accepted values: Alias, All, Function, Filter, Cmdlet, ExternalScript, Application, Script, **Workflow**

Add the 'Accepted values:` to the YAML block and add Workflow to the list above.

Reply:
Added the Accepted values to YAML and added Workflow to the list with brief description based on about_Workflows.